### PR TITLE
Bump rancher-webhook to v0.4.12-rc1

### PR DESCRIPTION
**Pull request by maxsokolovsky**

### CODE REVIEW

File: charts/rancher-webhook/103.0.10+up0.4.12-rc1/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.23.0-0 < 1.29.0-0'
+  catalog.cattle.io/namespace: cattle-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/rancher-version: '>= 2.8.0-0 < 2.9.0-0'
+  catalog.cattle.io/release-name: rancher-webhook
+apiVersion: v2
+appVersion: 0.4.12-rc1
```
Comment (tomleb) [inline charts/rancher-webhook/103.0.10+up0.4.12-rc1/Chart.yaml]: Shouldn't that be `0.4.12-rc.1`? We've moved away from this format, I see we had `0.4.11-rc.1`.

Comment (maxsokolovsky) [reply charts/rancher-webhook/103.0.10+up0.4.12-rc1/Chart.yaml]: Yup, I had noticed that right after pushing; updated.

Comment (maxsokolovsky) [reply charts/rancher-webhook/103.0.10+up0.4.12-rc1/Chart.yaml]: Yup, but I had pushed the webhook tag with the wrong format last week, and we want those immutable.
I noticed, though, that I had the wrong base version for the chart here, so updated.

Comment (tomleb) [reply charts/rancher-webhook/103.0.10+up0.4.12-rc1/Chart.yaml]: Right, so what I was suggesting was actually keeping the "wrong" format but still creating one with the right format and using that one. So in that case, you could create `0.4.12-rc.1` and use it, leaving `0.4.12-rc1` alone.

**Review by tomleb — APPROVED** [review]

**Review by prachidamle — APPROVED** [review]

**Pull request merged**